src/main/content/_includes/head.html

```diff
@@ -1,11 +1,19 @@
-<head>
+{% assign description = '' %}
```

Comment:
Please format the tabs in this block of new code.  The alignment of the code is not lined up and can be confusing what line is nested in what code block.